Neos.Flow/Classes/ObjectManagement/DependencyInjection/DependencyProxyTrait.php

```diff
@@ -0,0 +1,90 @@
+<?php
+namespace Neos\Flow\ObjectManagement\DependencyInjection;
+
+/*
+ * This file is part of the Neos.Flow package.
+ *
+ * (c) Contributors of the Neos Project - www.neos.io
+ *
+ * This package is Open Source Software. For the full copyright and license
+ * information, please view the LICENSE file which was distributed with this
+ * source code.
+ */
+
+trait DependencyProxyTrait
+{
+    /**
+     * @var string
+     */
+    protected $_dependencyClassName;
```

Comment:
would it make sense to make the properties private?

Reply:
I haven't tried it. But we need to careful, because proxy classes can be extended (by Flow).

In my opinion it doesn't make a difference if this was `private`, because only Flow can extend proxy classes.